hack/debug.sh

```diff
@@ -114,18 +114,10 @@ do_master () {
 	    continue
 	fi
 
-	if ! [[ $node =~ ^[0-9.]*$ ]]; then
-	    resolv_ip=$(awk '/\s'$node'$/ { print $1; exit; }' /etc/hosts)
-	    if [ -z "$resolv_ip" ]; then
-		resolv_ip=$(host $node 2>/dev/null | sed -ne 's/.*has address //p' | head -1)
-		if [ -z "$resolv_ip" ]; then
-		    echo "Node $node: no IP address in either DNS or /etc/hosts"
-		fi
-	    fi
+	resolv_ip=$(getent ahostsv4 $node | grep STREAM | awk '{ print $1}')
```

Comment:
can simplify to just `| awk '/STREAM/ { print $1 }'`

this will return junk if the host has multiple IP addresses, but the old code would too... but you could change `print $1` to `print $1; exit` to fix that